docassemble/juvenilesealing/data/questions/juvenile-sealing-informational-text.yml

```diff
@@ -398,8 +404,8 @@ content: |
 ---
 id: pdf_after_download_partial_v7_text
 template: pdf_after_download_partial_v7
+comment: Did this revert again? hr got removed. Did I never save it?
 content: |
-  
   ${hr}
```

Comment:
It looks like this did revert because when I was editing the file in the playground, this line was all `&#95;`s instead of `${hr}` and I had to change it back.